components/kyma-operator/cmd/operator/main.go

```diff
@@ -58,9 +58,21 @@ func main() {
 	backoffIntervalsRaw := flag.String("backoffIntervals", "10,20,40,60,80", "Number of seconds to wait before subsequent retries")
 	overrideLogFile := flag.String("overrideLogFile", STDOUT, "Log File to Print Installation overrides. (Default: /dev/stdout)")
 	overrideLogFormat := flag.String("overrideLogFormat", "text", "Installation Override Log format (Accepted values: text or json)")
+	helmMaxHistoryStr := flag.String("helmMaxHistory", "10", "Max number of releases returned by Helm release history query")
```

Comment:
Maybe just use https://golang.org/pkg/flag/#Int and https://golang.org/pkg/flag/#Int64 instead of casting?